backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/endpoint/EndpointResource.java

```diff
@@ -932,7 +932,7 @@ protected void redactSecretsForEndpoint(final SecurityContext securityContext, f
         boolean shouldRedactSecrets;
         if (this.backendConfig.isKesselRelationsEnabled(getOrgId(securityContext))) {
             try {
-                this.kesselAuthorization.hasViewPermissionOnResource(securityContext, IntegrationPermission.EDIT, ResourceType.INTEGRATION, endpoint.getId().toString());
```

Comment:
This one's interesting because it's an edit permission, but it's more of a speculative check- the user isn't changing anything, notifications just wants to know if the user hypothetically _could_ change something because, if so, they should see the unredacted secrets.

Reply:
Fair point, will change back.